tests/integration/callbacks/test_prevent_update.py

```diff
@@ -94,7 +95,10 @@ def show_clicks(n):
 
     dash_duo.start_server(app)
 
-    dash_duo.multiple_click("#btn", 10)
+    btn = dash_duo.wait_for_element("#btn")
+    for i in range(10):
+        btn.click()
+        time.sleep(0.2)
```

Comment:
Can we add this as maybe a `delay` kwarg to [`multiple_click`](https://github.com/plotly/dash/blob/f6abdfb823cdbe445eb7e660b29f1444e636a02a/dash/testing/browser.py#L527)? Seems like this would be useful more than once.

Reply:
Good idea.